### PR TITLE
Updates docker ee-repo with the new start and stop scripts.

### PR DESCRIPTION
**Pull request by kobalski**
Adds starts and stop scripts to docker image. Start script writes hazelcast PID to a file and stop scripts stops hazelcast instance started in docker container.

### CODE REVIEW

File: hazelcast-enterprise/server.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+if [ "x$MIN_HEAP_SIZE" != "x" ]; then
+	JAVA_OPTS="$JAVA_OPTS -Xms${MIN_HEAP_SIZE}"
+fi
+
+if [ "x$MAX_HEAP_SIZE" != "x" ]; then
+	JAVA_OPTS="$JAVA_OPTS -Xmx${MAX_HEAP_SIZE}"
+fi
+
+export CLASSPATH=$HAZELCAST_HOME/hazelcast-enterprise-$HZ_VERSION/lib/hazelcast-enterprise-all-3.5.3.jar
```
Comment (eminn) [inline hazelcast-enterprise/server.sh]: I think we should remove the hardcoded version from here.

Comment (bilalyasar) [inline hazelcast-enterprise/server.sh]: probably version will be updated from release script,

Comment (eminn) [inline hazelcast-enterprise/server.sh]: Don't we already have $HZ_VERSION placeholder for this ?

Comment (kobalski) [inline hazelcast-enterprise/server.sh]: I should use $HZ_VERSION. Thanks now it is correct. Release script should only update $HZ_VERSION.

**Pull request merged**